x-pack/plugins/observability/public/utils/format_alert_evaluation_value.ts

```diff
@@ -12,7 +12,7 @@ import {
 } from './get_alert_evaluation_unit_type_by_rule_type_id';
 
 export const formatAlertEvaluationValue = (ruleTypeId?: string, evaluationValue?: number) => {
-  if (!evaluationValue || !ruleTypeId) return '-';
+  if (evaluationValue === undefined || !ruleTypeId) return '-';
```

Comment:
This fixes the case when the threshold is set to 0. The rendered value was `-` when it should be the number